recipes/sortmerna/3.0.0/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: sortmerna
+  version: 3.0.0
+
+source:
+  url: https://github.com/biocore/sortmerna/releases/download/v3.0.0/sortmerna-3.0.0-linux.tar.gz
+  sha256: 8fbe5a579172a0908e50b1f5d8ba97b6e1e4509309605fb30eb5accb27024c19 
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
```

Comment:
zlib is exporting themself now, this is a new feature of the zlib package, so ideally you do not need to specify it anymore in the run dependency

Reply:
removed, thanks for reviewing :-)